apps/src/javalab/JavabuilderConnection.js

```diff
@@ -117,6 +122,7 @@ export default class JavabuilderConnection {
         break;
       case WebSocketMessageType.SYSTEM_OUT:
         this.onOutputMessage(data.value);
+        this.onNewlineMessage();
```

Comment:
Should this call `onNewlineMessage` be removed?

Reply:
Oh, good catch!  This was a last minute addition that caused those extra blank lines.